vignettes/VectorControl.Rmd

```diff
@@ -15,117 +15,124 @@ knitr::opts_chunk$set(
 ```
 
 ```{r setup}
-suppressPackageStartupMessages(library(ggplot2))
 library(malariasimulation)
 library(malariaEquilibrium)
-library(reshape2)
 ```
 
-# Parameterisation
+Long-lasting insecticidal bednets are highly effective and relatively cheap interventions to prevent malaria. The effects of insecticidal bed net distribution campsigns can be simulated using malariasimulation. The model provides the user with the flexibility to specify parameters that describe the net distribution campaign (e.g. timing, coverage, and target population), the bed nets distributed (e.g. efficacy, longevity), and the relative densities of the mosquito species present in the target population. We will illustrate this through an example with two bednet distributions, once per year.
```

Comment:
I'd drop the sentence on mosqutio species - there are a lot more things characteristing the vectors than just relative density, so probably not something to go into here